fw/http_tbl.c

```diff
@@ -499,6 +503,10 @@ tfw_cfgop_http_rule(TfwCfgSpec *cs, TfwCfgEntry *e)
 		rule->act.type = TFW_HTTP_MATCH_ACT_CACHE_TTL;
 		rule->act.cache_ttl = act_val_parsed;
 	}
+	else if (!strcasecmp(action, "jsch")) {
+		is_jsch_global = false;
```

Comment:
Here is inconsistent behavior - you change global variable only if one jsch is set.
For example:
config = """
http_chain {
    host == "aaa" -> jsch;
    host == "bbb" ->  good;
} 

If host == "aaa" ->jsch is set we don't support js challenge for "bbb". But if it is not set we support it! Why? How   host == "aaa" -> jsch; deal with host "bbb"?